paper-api/src/main/java/org/bukkit/event/entity/EntityPortalEvent.java

```diff
@@ -156,6 +161,30 @@ public int getCreationRadius() {
         return this.creationRadius;
     }
 
+    /**
+     * Sets the {@link WorldBorder} which the portal creation and search should be limited.
```

Comment:
```suggestion
     * Sets the {@link WorldBorder} which the portal search and optionally creation will
     * be limited to.
```

same for other setter and getters

Reply:
Would it be better to make getWorldBorder -> getWorldBorderOverride and return a nullable/optional? I'm starting to think its a little ugly to expose the world's worldborder here.

Reply:
thinking of it, adding api to create a virtual worldborder from an existing worldborder might be useful. that way we could also expose the default worldborder here and allow making changes to it without changing the actual worldborder.